docs/api/supervisor/endpoints.md

```diff
@@ -1406,6 +1406,8 @@ Get network information.
 | ---------- | ---------------------------------------------------------------------- |
 | interfaces | A list of [Network interface models](api/supervisor/models.md#network-interface) |
 | docker     | Information about the internal docker network |
+| host_connectivity | The connectivity state of the host network. |
```

Comment:
This could be clearer, what does the example value of `4` mean here? Is it a status code? The number of other clients in the network?

Reply:
Sure, It's the connectivity state